### PR TITLE
finished challenge

### PR DESCRIPTION
**Pull request by devonhackley**

### CODE REVIEW

**Comment by devonhackley** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code-challenges/401challenges/src/main/java/challenges/MergeSort.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            int[] b = Arrays.copyOfRange(arr, 0, middle);
+            int[] c = Arrays.copyOfRange(arr, middle, n);
+
+            //break down left side
+            mergeSort(b);
+
+            //break down right side
+            mergeSort(c);
+
+            W
```
Comment (evanslaton) [inline code-challenges/401challenges/src/main/java/challenges/MergeSort.java]: This prevents your code from compiling! Please remove this stray character!
